src/Uno.UI.Tests/Windows_UI_Xaml_Data/BindingTests/Given_Binding.cs

```diff
@@ -131,5 +131,25 @@ public void When_ElementName_In_Template_In_Template()
 
 			Assert.AreEqual(SUT.topLevel.Tag, tb.Text);
 		}
+
+		[TestMethod]
+		public void When_Xaml_Object_With_Common_Properties()
+		{
+			var SUT = new Binding_Xaml_Object_With_Common_Properties();
+
+			SUT.ForceLoaded();
+
+			Assert.AreEqual("42", SUT.topLevel.Text);
+		}
+
+		[TestMethod]
+		public void When_Xaml_Object_With_Xaml_Object_Properties()
+		{
+			var SUT = new Binding_Xaml_Object_With_Xaml_Object_Properties();
+
+			SUT.ForceLoaded();
+
+			Assert.AreEqual("42", SUT.topLevel.Text);
```

Comment:
My comment did not go through the first time... Apologies. 

You should be able to validate that all the properties from the binding are what you expect, using `GetBindingExpression` on `TextProperty`.